smdebug/tensorflow/keras.py

```diff
@@ -375,7 +390,7 @@ def _save_metrics(self, batch, logs, force_save=False):
             self._initialize_writers(only_initialize_if_missing=True)
             logs["batch"] = batch
             for key in logs:
-                if key in ["loss", "val_loss", "outputs"]:
+                if key in ["loss", "val_loss", "outputs", "y", "y_pred"]:
```

Comment:
y ypred is special name used at 374 378 and here. Can we make it as named constant?

Reply:
I have added constants